src/Applications/LDAS_App/util/config/generate_catchincr_hist.py

```diff
@@ -34,32 +44,32 @@
 template:    '%y4%m2%d2_%h2%n2z.bin',
 mode:        'instantaneous',
 frequency:   030000,
-ref_time:    000000,
-fields:     'TCFSAT_INCR'    , 'CATCHINCR'      ,
-            'TCFTRN_INCR'    , 'CATCHINCR'      ,
-            'TCFWLT_INCR'    , 'CATCHINCR'      ,
-            'QCFSAT_INCR'    , 'CATCHINCR'      ,
-            'QCFTRN_INCR'    , 'CATCHINCR'      ,
-            'QCFWLT_INCR'    , 'CATCHINCR'      ,
-            'CAPAC_INCR'     , 'CATCHINCR'      ,
-            'CATDEF_INCR'    , 'CATCHINCR'      ,
-            'RZEXC_INCR'     , 'CATCHINCR'      ,
-            'SRFEXC_INCR'    , 'CATCHINCR'      ,
-            'GHTCNT1_INCR'   , 'CATCHINCR'      ,
-            'GHTCNT2_INCR'   , 'CATCHINCR'      ,
-            'GHTCNT3_INCR'   , 'CATCHINCR'      ,
-            'GHTCNT4_INCR'   , 'CATCHINCR'      ,
-            'GHTCNT5_INCR'   , 'CATCHINCR'      ,
-            'GHTCNT6_INCR'   , 'CATCHINCR'      ,
-            'WESNN1_INCR'    , 'CATCHINCR'      ,
-            'WESNN2_INCR'    , 'CATCHINCR'      ,
-            'WESNN3_INCR'    , 'CATCHINCR'      ,
-            'HTSNNN1_INCR'   , 'CATCHINCR'      ,
-            'HTSNNN2_INCR'   , 'CATCHINCR'      ,
-            'HTSNNN3_INCR'   , 'CATCHINCR'      ,
-            'SNDZN1_INCR'    , 'CATCHINCR'      ,
-            'SNDZN2_INCR'    , 'CATCHINCR'      ,
-            'SNDZN3_INCR'    , 'CATCHINCR'      ,
+ref_time:    013000,
+fields:     'TCFSAT_INCR'    , 'CATCHINCR_e'      ,
+            'TCFTRN_INCR'    , 'CATCHINCR_e'      ,
+            'TCFWLT_INCR'    , 'CATCHINCR_e'      ,
+            'QCFSAT_INCR'    , 'CATCHINCR_e'      ,
+            'QCFTRN_INCR'    , 'CATCHINCR_e'      ,
+            'QCFWLT_INCR'    , 'CATCHINCR_e'      ,
+            'CAPAC_INCR'     , 'CATCHINCR_e'      ,
+            'CATDEF_INCR'    , 'CATCHINCR_e'      ,
+            'RZEXC_INCR'     , 'CATCHINCR_e'      ,
+            'SRFEXC_INCR'    , 'CATCHINCR_e'      ,
+            'GHTCNT1_INCR'   , 'CATCHINCR_e'      ,
+            'GHTCNT2_INCR'   , 'CATCHINCR_e'      ,
+            'GHTCNT3_INCR'   , 'CATCHINCR_e'      ,
+            'GHTCNT4_INCR'   , 'CATCHINCR_e'      ,
+            'GHTCNT5_INCR'   , 'CATCHINCR_e'      ,
+            'GHTCNT6_INCR'   , 'CATCHINCR_e'      ,
+            'WESNN1_INCR'    , 'CATCHINCR_e'      ,
+            'WESNN2_INCR'    , 'CATCHINCR_e'      ,
+            'WESNN3_INCR'    , 'CATCHINCR_e'      ,
+            'HTSNNN1_INCR'   , 'CATCHINCR_e'      ,
+            'HTSNNN2_INCR'   , 'CATCHINCR_e'      ,
+            'HTSNNN3_INCR'   , 'CATCHINCR_e'      ,
+            'SNDZN1_INCR'    , 'CATCHINCR_e'      ,
+            'SNDZN2_INCR'    , 'CATCHINCR_e'      ,
+            'SNDZN3_INCR'    , 'CATCHINCR_e'      ,
 """
 
 nens = 32
```

Comment:
this hardcoded input `nens = 32` should be moved to the top of the script, with a comment specifying that this is the number of ens members

Reply:
nens = 32 is the first line  of the  actual "code" ,a reasonable place to be.  It should not be among with  the declaration of "heads" and "hist_template" , which are like constant  lists or data tables. 

I pulled the py code with your edit commits,  it works for me and generates HISTORY.rc . 
I  added  back     label= """  :: """   and.   f.write(label) , for  a cleaner  separation between the collection block and the specification block in the generated HISTORY.rc  I run the 

I run the py and verifies the resulting HISTORY.rc. These two updated files ( py and HISTORY.rc.atmens ) are committed and pushed back to the branch.